communication/src/TlsWriteQueue.h

```diff
@@ -28,19 +28,22 @@
 #include "TlsDiagnostics.h"
 
 namespace bft::communication::tls {
-
+struct Header {
+  uint32_t msg_size;
+  NodeNum endpoint_num;
+};
```

Comment:
The struct should be packed:
#pragma pack(push, 1)
struct Header {
  uint32_t msg_size;
  NodeNum endpoint_num;
};
#pragma pack(pop)

Reply:
done. Thanks.